scripts/github-actions/tests-e2e-samples

```diff
@@ -18,16 +18,8 @@ set -o nounset
 set -o pipefail
 
 REPO_ROOT="$(git rev-parse --show-toplevel)"
-
 cd ${REPO_ROOT}/
 
-echo "Downloading envtest assets..."
-export KUBEBUILDER_ASSETS=$(go run sigs.k8s.io/controller-runtime/tools/setup-envtest@latest use -p path)
-
-echo "Running fixtures in tests/e2e..."
+echo "Running samples in tests/e2e..."
 
-RUN_E2E=1 \
-E2E_KUBE_TARGET=envtest \
-E2E_GCP_TARGET=mock \
-KCC_USE_DIRECT_RECONCILERS="SQLInstance,ComputeForwardingRule" \
-  go test -test.count=1 -timeout 3600s -v ./tests/e2e -run TestAllInSeries/samples
\ No newline at end of file
+RUN_TESTS=TestAllInSeries/samples dev/tasks/run-e2e
```

Comment:
I feel `dev/tasks/run-e2e` and the original command should be the same, not sure why the original command stopped working...

Is that similar to the issue addressed in #2608?